protection.lua

```diff
@@ -90,14 +90,39 @@ minetest.register_craftitem("simple_protection:claim", {
 minetest.register_alias("simple_protection:claim_stick", "simple_protection:claim")
 minetest.register_alias("claim_stick", "simple_protection:claim")
 
-minetest.register_craft({
-	output = "simple_protection:claim",
-	recipe = {
-		{"default:copper_ingot", "default:steel_ingot", "default:copper_ingot"},
-		{"default:steel_ingot", "default:stonebrick", "default:steel_ingot"},
-		{"default:copper_ingot", "default:steel_ingot", "default:copper_ingot"},
-	}
-})
+-- If neither one of these occurs then the claim stick isn't craftable
+if minetest.registered_items["default:dirt"] then
```

Comment:
Use mod names to distinguish games. Perhaps move such variable into `init.lua` for mod-wide access?

Reply:
Yup, where it should be.